include/smack/SmackRep.h

```diff
@@ -150,6 +166,14 @@ class SmackRep {
   virtual const Stmt* alloca(llvm::AllocaInst& i);
   virtual const Stmt* memcpy(const llvm::MemCpyInst& msi);
   virtual const Stmt* memset(const llvm::MemSetInst& msi);
+  virtual const Stmt* loadAsBytes(const llvm::LoadInst& li);
+  virtual const Stmt* storeAsBytes(const llvm::StoreInst& si);
```

Comment:
A issue with the proposed methods is that in order to get the safety information of the memory locations loaded/stored, the methods need to know the function where the instruction is. I'm not sure if we could get the pointer to the function via addr variable. I guess the chance is low since the interface, TypeSafety::TypeSafe requires an argument of llvm::Function* which is probably unnecessary if it could be obtained from another argument of type llvm::Value. 

Moreover, the two methods are called at very limited times. So I was wondering if we could stick to the current implementation.